src/chart/Pie.js

```diff
@@ -84,6 +58,8 @@
             this._palette = this._palette.cloneNotExists(this.paletteID() + "_" + this.id());
         }
 
+        this.d3Arc.innerRadius(this.width() * this.innerRadius() ? this.innerRadius() : 0);
```

Comment:
@GordonSmith that is where the innerradius (pie whole is adjusted) ... if its not updated there it isn't updated correctly ... we had discussed that it needs to go in the update function ... the radius param is now a % as in % of the width of the circle (i dont think i need to divide by 2 do i?)